web/src/beta/components/SidePanelSectionField/index.tsx

```diff
@@ -28,7 +28,7 @@ const SidePanelSectionField: React.FC<{
 };
 
 const Field = styled.div`
-  background: ${({ theme }) => theme.bg[1]};
+  background: ${({ theme }) => theme.bg[0]};
```

Comment:
This messes up the background of other components that use this. Please revert this and wrap SidePanelSectionField as a styled component to change the background color 🙏🏻